107/src/main/java/org/ehcache/jsr107/Eh107CacheLoaderWriter.java

```diff
@@ -28,26 +28,31 @@
 
 import javax.cache.integration.CacheLoader;
 import javax.cache.integration.CacheWriter;
-import org.ehcache.spi.loaderwriter.BulkCacheWritingException;
 
-import org.ehcache.spi.loaderwriter.CacheLoaderWriter;
+import org.ehcache.jsr107.internal.JsrCacheLoaderWriter;
+import org.ehcache.spi.loaderwriter.BulkCacheLoadingException;
+import org.ehcache.spi.loaderwriter.BulkCacheWritingException;
 
 /**
  * @author teck
  */
-class Eh107CacheLoaderWriter<K, V> implements CacheLoaderWriter<K, V>, Closeable {
+class Eh107CacheLoaderWriter<K, V> implements JsrCacheLoaderWriter<K, V>, Closeable {
 
   private final CacheLoader<K, V> cacheLoader;
+  private final boolean skipLoader;
```

Comment:
I think the logic in this class would read clearer is we kept this as `readThrough`